astro/src/content/docs/release-notes/index.mdx

```diff
@@ -38,6 +38,21 @@ import IssueResolvedVia from 'src/components/docs/release-notes/ReleaseNoteIssue
 
 Looking for release notes older than 1.23.0? Look in the [release notes archive](/docs/release-notes/archive). Looking to be [notified of new releases?](/docs/operate/roadmap/releases#release-notifications) <span class="not-prose no-underline"><a class="ml-2" href="/docs/releases.xml"><i class="fas fa-xs fa-rss text-orange-700 text-2xl" width="50px" /></a></span>
 
+<ReleaseNoteHeading version='1.53.3' releaseDate='October 22, 2024'/>
+
+### Fixed
+<Issue issue="2764">
```

Comment:
Do the `Issue` components format correctly when used next to each other and mixed with `.mdx`? I’m not able to check right now.

Reply:
I think the proper usage is
```
* <Issue issue="2764">slot text</Issue>
```

Reply:
The formatting within the component seems good, but the release notes lines using the component need to be bulleted as noted above.

Reply:
Fixed any rendering issues. I was trying to get it to render a numbered list in a bullet but that's going to end up being more complicated than expected.